s12/protobuf/proto/validator.proto

```diff
@@ -214,6 +214,11 @@ message IdRules {
   // the beginning of the string.
   // @TODO: implement if we need this, e.g. for user_UUID, org_UUID or use a dedicated validation type for this
   // optional string prefix = 5;
+
+  // Only apply soft validation. Validation errors for this validator are logged, not returned.
+  // Be aware that due to multiples validators applying, and order of precedence
+  // this will not prevent errors from other validators being returned.
+  optional bool soft_validation = 5 [default = false];
```

Comment:
Thanks @laurence-smith, what do you think about the suggestion to rename this to `monitor_mode` or `report_only` or `log_only`?
It's a good idea to test validation for existing fields first so we could add this to other validators as well later. Can you please also update the README.md?

Reply:
log_only is a sensitive solution